posts/ja/2023-01-30-22.0.0.11_ja.adoc

```diff
@@ -0,0 +1,467 @@
+---
+layout: post
+title: "Java SE 19 and distributed security cache support in Open Liberty 22.0.0.11"
```

Comment:
`title` needs to be translated from `en` to `ja`

Reply:
Hi @una-tapa, I have used machine translation for the suggested change below. Can you help me double check if the translation is appropriate?
```suggestion
title: "Open Liberty での Java SE 19 および分散セキュリティー・キャッシュのサポート 22.0.0.11"
```